packages/valaxy-theme-yun/components/YunPageHeader.vue

```diff
@@ -3,11 +3,19 @@ defineProps<{
   color?: string
   icon?: string
   title?: string
+  cover?: string
```

Comment:
I don't think cover needs to be passed in to the pageheader.

And the style needs to be modified.

![image](https://user-images.githubusercontent.com/25154432/173239694-b92d339f-3c9b-480d-9345-b9d32a66ca13.png)

I think a beautiful design should be placed in the red frame.